cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMakerCluster.java

```diff
@@ -133,8 +133,8 @@ public static KafkaMirrorMakerCluster fromCrd(KafkaMirrorMaker kafkaMirrorMaker,
         KafkaMirrorMakerCluster kafkaMirrorMakerCluster = new KafkaMirrorMakerCluster(kafkaMirrorMaker);
 
         KafkaMirrorMakerSpec spec = kafkaMirrorMaker.getSpec();
-        kafkaMirrorMakerCluster.setReplicas(spec != null && spec.getReplicas() > 0 ? spec.getReplicas() : DEFAULT_REPLICAS);
         if (spec != null) {
```

Comment:
To be honest, I am not sure, if we need this check (`spec != null`) here ? For example in `KafkaConnectCluster` and in `KafkaBridgeCluster`  `spec` is not checked. (if it is null or not)

Reply:
I guess you can do what the other classes do.